week01/Majority_Element.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    # TC: O(n), MC: O(n)
+    def majorityElement(self, nums: List[int]) -> int:
+        dict = {}
+        
+        for i in range(len(nums)):
+            if nums[i] in dict.keys():
+                dict[nums[i]] += 1
+            else:
+                dict[nums[i]] = 1
+        
+        maximum = 0
+        element = nums[0]
+        for ele in dict:
+            if dict[ele] > maximum:
+                maximum = dict[ele]
+                element = ele
+                
+        return element
```

Comment:
Not an acceptable solution for this problem. This is pure brute force. The interviewer will obviously look for something different if he/she asks you about this problem.

QQ: Did you attend our classes?